data-store-api/resources/schema.json

```diff
@@ -181,7 +181,7 @@
                     "$id": "#/properties/publisher_id",
                     "type": "string",
                     "title": "Publisher",
-                    "description": "Please provide information about the organisation which published/produced this dataset. If your organisation produced this dataset, please select it again using the tool below."
+                    "description": "Please provide information about the organisation which publishing/producing this dataset. If this is your organisation, please select it again using the tool below."
```

Comment:
"which publishing/producing" is weird grammar. maybe

Please provide information about the organisation which is publishing or producing this dataset. If this is your organisation, please select it again using the tool below.

Same for other change.

Reply:
Agree I miss the is. Will change / to "or" as well :)